specifyweb/specify/api.py

```diff
@@ -252,6 +277,36 @@ def collection_dispatch(request, model) -> HttpResponse:
 
     return resp
 
+def collection_dispatch_bulk(request, model) -> HttpResponse:
+    """
+    Do the same as collection_dispatch, but for bulk POST operations.
+    Call this endpoint with a list of objects of the same type to create.
+    This reduces the amount of API calls needed to create multiple objects, like when creating multiple carry forwards.
+    """
+    resp: HttpResponse
+    checker = table_permissions_checker(request.specify_collection, request.specify_user_agent, "read")
+
+    if request.method == 'POST':
+        data = json.load(request)
```

Comment:
Not sure if the two are semantically equivalent (it looks to be the case), but I would rather use a `json.loads(request.body)` here instead of `json.load`. 

https://docs.djangoproject.com/en/5.0/ref/request-response/#django.http.HttpRequest.body. 

Or by using `json.load` do we also handle the case where the client does not use a traditional body and instead uses form data?